src/lib.js

```diff
@@ -574,6 +574,14 @@ async function getItems(base64, customTextures = false, packs, cacheOnly = false
       }
     }
 
+    if (helper.hasPath(item, "tag", "ExtraAttributes", "blocks_walked")) {
+      let { blocks_walked } = item.tag.ExtraAttributes;
+
+      if (blocks_walked > 0) {
+        item.extra.blocks_walked = blocks_walked;
+      }
+    }
+
```

Comment:
Why do we need this section can't we just use the value from the tag when we need it later on instead of storing redundant information in extra

Reply:
It's the same way expertise kills are done (and other things too iirc)... shouldn't be too much of an issue.

Normally item.extra has a "parsed" version of item.tag.ExtraAttributes, but we usually just store everything there, even if no parsing is required.

Reply:
> storing redundant information in extra
how is "blocks_walked" redundant info? It's basically the same procedure like for the expertise_kills. If I delete the lines you are reffering to the added lore does not show at all... Couls you clarify what you mean by "why do we need this secion". If you mean what I think you do then we don't need the corresponding lines of code for the other tags - like expertise_kills, rarity_upgrades, etc. - either. But then the nice and tidy seperation between the information extraction and the lore creation gets broken up.

Reply:
> how is "blocks_walked" redundant info

It's redundant because it is already stored in tag so it doesn't really need to be stored in extra

However if it is the same as `expertise_kills` then it should be fine we can refactor it later but we should stay consistent by keeping similar systems codded similarly

Reply:
Yeah how he did it it's consistent with current code 👍 

And yes it could use a refactor.... together with pretty much half of lib.js 😛